mygithub/src/components/github/Profile.js

```diff
@@ -1,58 +1,59 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import RepoList from './RepoList.js';
-class Profile extends Component{
-	
-	render(){
-// Nesting profile component in App component
-		return(
+class Profile extends Component {
+
+	render() {
+		// Nesting profile component in App component
+		return (
 			<div>
 				<div className="panel panel-default">
-					  <div className="panel-heading"></div>
-					  <div className="panel-body">
-					    <h1> Name :{this.props.userData.name} </h1>
-					  </div>
+					<div className="panel-heading"></div>
+					<div className="panel-body">
+						<h1> Name :{this.props.userData.name} </h1>
+						<h1> Name :{this.props.userData.address} </h1>
```

Comment:
You are now introducing a new field `address` in this PR. Can you elaborate on its purpose and how it relates to the user's GitHub profile?

For consistency and to avoid confusion, it's a good practice to stick with the existing GitHub API field names like `location` to represent a user's whereabouts.